examples/flower_example.rs

```diff
@@ -0,0 +1,35 @@
+use polars::frame::DataFrame;
+use polars::series::Series;
+use polars::error::PolarsResult;
+use polars_core::prelude::*;
+use polars::prelude::NamedFrom;
+use polars::datatypes::Float64Type;
```

Comment:
Any reason you prefer separate lines instead of 
```rust
use polars::{
    datatypes::Float64Type;
    error::PolarsResult,
    frame::DataFrame,
    series::Series,
};
```
?

Reply:
No that's better!